test/optimizations/autoLocalAccess/regularDeclaration.chpl

```diff
@@ -27,7 +27,7 @@ use common;
   B = 3;
   C = 7;
 
-  // this only optimizes `A[i]` can do more static tracing to optimize others
```

Comment:
Is this change due to this PR, or just a housekeeping mod that got noticed at the same time?

Reply:
The latter. We have been able to optimize all three of them statically since the first automatic local access PR.

Most likely, I have implemented that support later in my effort, and forgot to remove the comment.